src/app/backend/dashboard.go

```diff
@@ -56,10 +56,15 @@ var (
 		"to connect to in the format of protocol://address:port, e.g., "+
 		"http://localhost:8080. If not specified, the assumption is that the binary runs inside a "+
 		"Kubernetes cluster and local discovery is attempted.")
-	argHeapsterHost = pflag.String("heapster-host", "", "The address of the Heapster Apiserver "+
+	argMetricsProvider = pflag.String("metrics-provider", "sidecar", "Select provider type for metrics. Defaults to 'sidecar'")
```

Comment:
Don't include the default in the help text, this is already printed by the help system:
```
      --metrics-provider string          Select provider type for metrics. Defaults to 'sidecar' (default "sidecar")
```
Consider instead listing the options: "sidecar" or "heapster".

Reply:
I'm picking this comment to say: You're awesome and I'm going to try to go through all these in the next couple days. Thanks so much!

Reply:
Will you change that?

Reply:
I totally missed this, and I'll PR in a change this week.